lib/chef/azure/commands/enable.rb

```diff
@@ -173,6 +173,7 @@ def configure_chef_only_once
         config[:node_ssl_verify_mode] =  bootstrap_options['node_ssl_verify_mode'] if bootstrap_options['node_ssl_verify_mode']
         runlist = @run_list.empty? ? [] : escape_runlist(@run_list)
         load_cloud_attributes_in_hints if ! @ohai_hints.empty?
+        config[:first_boot_attributes] = eval(@first_boot_attributes)

```

Comment:
Why `eval`? Is there ruby code in this variable? It seems unlikely. If there is, we need more clarity on what goes into this variable.